tgui-next/packages/tgui/styles/themes/syndicate.scss

```diff
@@ -12,6 +12,10 @@
   $border-radius: 2px,
 );
 
+@use '../components/Input.scss' with (
```

Comment:
I believe you fucked up the default theme, because themed imports must be behind the theme class.